src/list/vm.js

```diff
@@ -115,8 +125,19 @@ module.exports = function createList(config) {
 		});
 	}
 
+	var defaultOrderIdx;
+
+	if (orderField) {
+		defaultOrderIdx = findSortIdx(config.orderBy);
+	}
+
 	var sort = ko.observable(sortOptions[defaultOrderIdx || 0]);
-	var sortIdx = defaultOrderIdx || 0;
+
+	var sortIdx = ko.computed(function() {
+		var sortVal = sort().value;
+
+		return findSortIdx(sortVal);
+	}, this);
```

Comment:
this means the window object here. You don't need to pass it, since you don't use it in the function you passed to the computed.